### PR TITLE
Deprecate old auditing cmdlets

### PR DESCRIPTION
**Pull request by bashahee**
## Description

Deprecating old SQL Auditing cmdlets, since breaking changes are allowed to be introduced in the upcoming release.
Moreover, tests of old cmdlets were removed as well.

## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Sql/Sql/Auditing/Model/ServerAuditModel.cs [hunk 1]
```diff
@@ -19,6 +19,34 @@
...
+        SCHEMA_OBJECT_ACCESS_GROUP,
+        SCHEMA_OBJECT_CHANGE_GROUP,
+        SCHEMA_OBJECT_OWNERSHIP_CHANGE_GROUP,
+        SCHEMA_OBJECT_PERMISSION_CHANGE_GROUP,
+        SUCCESSFUL_DATABASE_AUTHENTICATION_GROUP,
+        USER_CHANGE_PASSWORD_GROUP
+    }
+
+    public enum StorageKeyKind { Primary, Secondary };
+
```
Comment (wyunchi-ms) [inline src/Sql/Sql/Auditing/Model/ServerAuditModel.cs]: It seems that these variable is not referenced. Maybe you can add this modification in other commit.

Comment (bashahee) [reply src/Sql/Sql/Auditing/Model/ServerAuditModel.cs]: @wyunchi-ms 
They are referenced in unmodified files, like SetAzSqlDatabaseAudit.cs.

**Comment by bashahee** [comment]
@erich-wang I am not sure what is the reason of the failures in (Test linux).

**Comment by wyunchi-ms** [comment]
Hi @bashahee there are some conflicts. Could you please take a look at this?

**Comment by bashahee** [comment]
@wyunchi-ms done

**Pull request merged**